File: app/src/main/java/com/hfad/beeradviser/Java/BeerExpert.java
package com.hfad.beeradviser.Java;

import java.util.ArrayList;
import java.util.List;

public class BeerExpert {
    List<String> getBrands(String color) {
        List<String> brands = new ArrayList<>();
        if (color.equals("янтарный")) {
            brands.add("Джек Амбер");
            brands.add("Красный лось");
        } else {
            brands.add("Тюрьма Бледный Эль");
            brands.add("Сгусток Толстый");
        }
        return brands;
    }
}

File: app/src/main/java/com/hfad/beeradviser/Kotlin/BeerExpert.kt
package com.hfad.beeradviser.Kotlin

import java.util.ArrayList

class BeerExpert {
    fun getBrands(color: String): List<String> {
        val brands = arrayListOf<String>()
        brands.apply {
            if (color == "янтарный") {
                add("Джек Амбер")
                add("Красный лось")
            } else {
                add("Тюрьма Бледный Эль")
                add("Сгусток Толстый")
            }
        }
        return brands
    }
}
